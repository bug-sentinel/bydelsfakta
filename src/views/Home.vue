<template>
  <h1>Forside</h1>
</template>

<script>
export default {
  name: 'Home',
};
</script>
