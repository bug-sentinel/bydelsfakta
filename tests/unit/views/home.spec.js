import { mount } from '@vue/test-utils';
import Home from '../../../src/views/Home.vue';

describe('Home', () => {
  let wrapper = null;

  beforeEach(() => {
    wrapper = mount(Home);
  });

  afterEach(() => {
    wrapper.destroy();
  });

  test('Renders home-component and finds h1 with forside', () => {
    expect(wrapper.find('h1').is('h1')).toBe(true);
  });

  test('Home renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
