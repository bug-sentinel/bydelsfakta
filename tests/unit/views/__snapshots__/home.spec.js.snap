// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Home renders correctly 1`] = `
<h1>
  Forside
</h1>
`;
